policyengine_us/variables/gov/states/id/tax/income/credits/grocery_credit/id_grocery_credit.py

```diff
@@ -0,0 +1,37 @@
+from policyengine_us.model_api import *
+
+
+class id_grocery_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Idaho grocery credit"
+    unit = USD
+    definition_period = YEAR
+    defined_for = "id_grocery_credit_eligible"
+
```

Comment:
Split out the variables to support this
```suggestion
    adds = ["id_base_grocery_credit", "id_enhanced_grocery_credit", "id_aged_grocery_credit"]
```